cobbler/items/distro.py

```diff
@@ -287,12 +290,10 @@ def initrd(self, initrd: str):
         """
         if not isinstance(initrd, str):
             raise TypeError("initrd must be of type str")
-        if not initrd:
-            raise ValueError("initrd not specified")
-        if utils.find_initrd(initrd):
-            self._initrd = initrd
-            return
-        raise ValueError("initrd not found")
+        if initrd:
+            if not utils.find_initrd(initrd):
+                raise ValueError("initrd not found: %s" % initrd)
+        self._initrd = initrd
```

Comment:
@olafhering I had to rebase your PR. Can you please double check if these changes are the ones you intended or did I mess that up during the rebase?

Reply:
It looks ok to me.